packages/configs/README.md

```diff
@@ -22,11 +22,11 @@ There are currently 3 types of packages that Amplify UI supports:
 
 ## Typescript
 
-To prevent drift between the `compilerOptions` of the _tsconfig.json_ files in the Amplify UI monorepo as more packages are added, base Typescript `compilerOptions` configurations are stored within the [ts](./ts) directory. The **[Base Config](./ts/tsconfig.base.json)** is extended directly or indirectly in all platform and utility packages.
+To prevent drift between the `compilerOptions` of the _tsconfig.json_ files in the Amplify UI monorepo as more packages are added, base Typescript `compilerOptions` configurations are stored within the [ts](./typescript) directory. The **[Base Config](./typescript/tsconfig.base.json)** is extended directly or indirectly in all platform and utility packages.
```

Comment:
```suggestion
To prevent drift between the `compilerOptions` of the _tsconfig.json_ files in the Amplify UI monorepo as more packages are added, base Typescript `compilerOptions` configurations are stored within the [typescript](./typescript) directory. The **[Base Config](./typescript/tsconfig.base.json)** is extended directly or indirectly in all platform and utility packages.
```